joplin/base/wagtail_hooks.py

```diff
@@ -79,26 +79,33 @@ def joplin_page_listing_more_buttons(page, page_perms, is_parent=False):
             attrs={'title': _("Copy page '{title}'").format(title=page.get_admin_display_title())},
             priority=20
         )
-    if page_perms.can_delete():
-        yield Button(
-            _('Delete'),
-            reverse('wagtailadmin_pages:delete', args=[page.id]),
-            attrs={'title': _("Delete page '{title}'").format(title=page.get_admin_display_title())},
-            priority=30
-        )
+    # if not page.live:
```

Comment:
This removes the delete option, we don't have any archive functionality yet but once we do we can uncomment this and work towards an archive page flow

Reply:
I agree that it could be slightly problematic, but I think we should address it by making https://github.com/cityofaustin/techstack/issues/694 (implementing archived status) a priority.